src/main/java/com/knighttodo/knighttodo/service/impl/BlockServiceImpl.java

```diff
@@ -1,36 +1,42 @@
 package com.knighttodo.knighttodo.service.impl;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
 import com.knighttodo.knighttodo.domain.BlockVO;
 import com.knighttodo.knighttodo.domain.RoutineVO;
 import com.knighttodo.knighttodo.domain.TodoVO;
```

Comment:
whyyyyy did you change an order of the imports?)

Reply:
sorry, forgot to move them back. You should create your own auto code style to make it work as you want ) I'll try to fix it somehow if you explain to me one more time how should it be because I don't have any issues with their order and don't really understand what is the difference between it.

Reply:
Is it not a big deal, but this variant is strange for me because they are not sorted alphabetically. I use the same editor config and Intellij always sorts it))

Reply:
sorry, forgot to move them back. You should create your own auto code style to make it work as you want ) I'll try to fix it somehow if you explain to me one more time how should it be, because I don't have any issues with their order and don't really understand what is the difference between it.